store.go

```diff
@@ -578,8 +578,8 @@ type AdditionalLayer interface {
 	// layer store.
 	PutAs(id, parent string, names []string) (*Layer, error)
 
-	// UncompressedDigest returns the uncompressed digest of this layer
-	UncompressedDigest() digest.Digest
+	// TOCDigest returns the digest of TOC of this layer.
```

Comment:
https://github.com/containers/image/pull/2416 suggests that this can return `""`. If so, that should be documented here.

Or, alternatively, the code should promise (and check?) that that’s not the case.

Reply:
Added the comment.